### PR TITLE
fix: Markdownlint

### PR DESCRIPTION
**Pull request by nschonni**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md [hunk 1]
```diff
@@ -23,35 +23,36 @@ helpviewer_keywords:
...
-### To place multiple statements on the same line  
-  
-- Separate the statements with a colon (`:`), as in the following example.  
-  
-     [!code-vb[VbVbcnConventions#10](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbcnConventions/VB/Class1.vb#10)]  
-  
+
+When writing your code, you might at times create lengthy statements that necessitate horizontal scrolling in the Code Editor. Although this doesn't affect the way your code runs, it makes it difficult for you or anyone else to read the code as it appears on the monitor. In such cases, you should consider breaking the single long statement into several lines.
+
+### To break a single statement into multiple lines
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md]:
```suggestion
## To break a single statement into multiple lines
```

File: docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md [hunk 2]
```diff
@@ -23,35 +23,36 @@ helpviewer_keywords:
...
+
+  The line-continuation character must be the last character on a line. You can't follow it with anything else on the same line.
+
+  Some limitations exist as to where you can use the line-continuation character; for example, you can't use it in the middle of an argument name. You can break an argument list with the line-continuation character, but the individual names of the arguments must remain intact.
+
+  You can't continue a comment by using a line-continuation character. The compiler doesn't examine the characters in a comment for special meaning. For a multiple-line comment, repeat the comment symbol (`'`) on each line.
+
+ Although placing each statement on a separate line is the recommended method, Visual Basic also allows you to place multiple statements on the same line.
+
+### To place multiple statements on the same line
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md]:
```suggestion
## To place multiple statements on the same line
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good. Do you mind applying the suggestions I called out?

File: docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md [hunk 3]
```diff
@@ -23,35 +23,36 @@ helpviewer_keywords:
...
-- Separate the statements with a colon (`:`), as in the following example.  
-  
-     [!code-vb[VbVbcnConventions#10](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbcnConventions/VB/Class1.vb#10)]  
-  
+
+When writing your code, you might at times create lengthy statements that necessitate horizontal scrolling in the Code Editor. Although this doesn't affect the way your code runs, it makes it difficult for you or anyone else to read the code as it appears on the monitor. In such cases, you should consider breaking the single long statement into several lines.
+
+### To break a single statement into multiple lines
+
+- Use the line-continuation character, which is an underscore (`_`), at the point at which you want the line to break. The underscore must be immediately preceded by a space and immediately followed by a line terminator (carriage return) or (starting with version 16.0) a comment followed by a carriage return.
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md]: I also noticed that these bullets under each section is sort of unnecessary.

Comment (nschonni) [reply docs/visual-basic/programming-guide/program-structure/how-to-break-and-combine-statements-in-code.md]: Yeah, think I saw a few like this before. I figured it was an automation/templating thing so I just ignored them :)

**Comment by nschonni** [comment]
Rebased for a few stray MD032 issues after the other PR landed

File: docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md [hunk 4]
```diff
@@ -67,7 +69,9 @@ Hello World!
 Press any key to close this window . . .
 ```
 
-Press a key to end the session. 
+Press a key to end the session.
+
+---
```
Comment (mairaw) [inline docs/csharp/programming-guide/inside-a-program/hello-world-your-first-program.md]: Good catch!

File: docs/csharp/programming-guide/interop/walkthrough-office-programming.md [hunk 5]
```diff
@@ -199,11 +199,7 @@ You must have Microsoft Office Excel and Microsoft Office Word installed on your
 - [Lambda Expressions (C#)](../statements-expressions-operators/lambda-expressions.md)
 - [How to: Use Indexed Properties in COM Interop Programming](./how-to-use-indexed-properties-in-com-interop-rogramming.md)
 - [Walkthrough: Embedding Type Information from Microsoft Office Assemblies in Visual Studio](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120))
-<<<<<<< HEAD
```
Comment (mairaw) [inline docs/csharp/programming-guide/interop/walkthrough-office-programming.md]: ouch... let me double check we don't have more of those in our docs after that giant PR was merged.

Comment (nschonni) [reply docs/csharp/programming-guide/interop/walkthrough-office-programming.md]: I did a quick search for `<<<<` and that was the only one I found
Feel free to land/fix this sepately if you want to just fix the page

File: docs/framework/configure-apps/how-to-create-a-publisher-policy.md [hunk 6]
```diff
@@ -9,87 +9,90 @@ helpviewer_keywords:
...
 
 Another consequence is that the version 2.0 linker cannot be used to create a publisher policy assembly for an assembly compiled using earlier versions of the .NET Framework, because it always specifies processor architecture.
 
-## Adding the Publisher Policy Assembly to the Global Assembly Cache  
+## Adding the Publisher Policy Assembly to the Global Assembly Cache
+
 Use the [Global Assembly Cache tool (Gacutil.exe)](../tools/gacutil-exe-gac-tool.md) to add the publisher policy assembly to the global assembly cache.
 
-#### To add the publisher policy assembly to the global assembly cache  
+#### To add the publisher policy assembly to the global assembly cache
```
Comment (mairaw) [inline docs/framework/configure-apps/how-to-create-a-publisher-policy.md]:
```suggestion
### To add the publisher policy assembly to the global assembly cache
```

File: includes/core-changes/vbnewline-is-obsolete.md [hunk 7]
```diff
@@ -22,13 +22,5 @@ The [Obsolete](xref:System.ObsoleteAttribute) attribute message for `vbNewLine`
 Visual Basic
 
 #### Affected APIs
-- <xref:Microsoft.VisualBasic.Constants.vbNewLine?displayProperty=fullName>
-
-<!--
```
Comment (mairaw) [inline includes/core-changes/vbnewline-is-obsolete.md]: I remember @rpetrusha telling me this is needed for some tooling on the breaking change PR. Please restore.

https://github.com/dotnet/docs/pull/14001#discussion_r322988531

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you for making the additional changes @nschonni. Left two additional comments.

File: docs/framework/app-domains/build-single-file-assembly.md [hunk 8]
```diff
@@ -80,6 +79,7 @@ csc -out:myCodeLibrary.dll -t:library myCode.cs
 ```vb
```
Comment (nschonni) [inline docs/framework/app-domains/build-single-file-assembly.md]: Aside, I noticed a few of these compiler commanline blocks had language codes, but I left them in.
Probably good to to a find of ```vb\n\s*vbc and the same for ```csharp\n\s*csc

File: includes/core-changes/vbnewline-is-obsolete.md (hunk 7 above)
Comment (nschonni) [reply includes/core-changes/vbnewline-is-obsolete.md]: Added the comment section back

File: docs/csharp/programming-guide/interop/walkthrough-office-programming.md (hunk 5 above)
Comment (mairaw) [reply docs/csharp/programming-guide/interop/walkthrough-office-programming.md]: That's good. Thanks for checking!

File: docs/framework/app-domains/build-single-file-assembly.md (hunk 8 above)
Comment (mairaw) [reply docs/framework/app-domains/build-single-file-assembly.md]: The problem is that they want a specific console command for each language. Not really sure how to solve these now. If you change them to be console and not a language, then we need to indicate the language for each command.

**Review by mairaw — APPROVED** [review]
LGTM

**Pull request merged**